src/uu/chmod/src/chmod.rs

```diff
@@ -195,21 +195,24 @@ impl Chmoder {
                         filename.quote()
                     );
                     if !self.quiet {
-                        return Err(USimpleError::new(
+                        show!(USimpleError::new(
                             1,
                             format!("cannot operate on dangling symlink {}", filename.quote()),
                         ));
                     }
+                    set_exit_code(1);
+                    continue;
                 } else if !self.quiet {
-                    return Err(USimpleError::new(
+                    show!(USimpleError::new(
                         1,
                         format!(
                             "cannot access {}: No such file or directory",
                             filename.quote()
-                        ),
+                        )
                     ));
                 }
-                return Err(ExitCode::new(1));
+                set_exit_code(1);
```

Comment:
Could you add a small comment explaining that this is necessary for the case where `self.quiet` is `true`?

Reply:
I just added it because noticed GNU did it :stuck_out_tongue: (but added the comment anyway)

Reply:
Oh yeah I understand you're just matching the behaviour! I was thinking a comment along these lines:
```
// GNU exits with exit code 1 even if -q or --quiet are passed
// So we set the exit code, because it hasn't been set yet if `self.quiet` is true.
```

Reply:
Ah, my bad. Basically copy pasted your comment on the code. This should have been a straightforward PR, yet I have managed to make it difficult :P

Reply:
No worries!